composer.json

```diff
@@ -13,7 +13,7 @@
     "require": {
         "php": ">=5.3.4",
         "typo3/cms-core": ">=6.2.14",
-        "luracast/restler": "3.0.0.20160315"
+        "luracast/restler": "3.0.0-RC6"
```

Comment:
Because "luracast/restler" does not have any up-to-date tags, we try to regularly update our fork (from the v3 branch), create a tag, test and make a release. 

Did you notice any issues so far?